phpcs.xml

```diff
@@ -20,19 +20,15 @@
 	<rule ref="Generic.CodeAnalysis.UnnecessaryFinalModifier"/>
 	<!-- Do not override methods to call their parent -->
 	<rule ref="Generic.CodeAnalysis.UselessOverridingMethod"/>
-	<!-- One line should not have more than 80 characters -->
-	<!-- One line must never exceed 120 characters -->
+	<!-- Maximum line length -->
 	<rule ref="Generic.Files.LineLength">
 		<!-- For language strings maximum line lengths make little sense. -->
 		<exclude-pattern>./app/i18n/</exclude-pattern>
 		<!-- Don't enforce line length on the HTML; the point is to improve legibility, not reduce it -->
-		<exclude-pattern>./app/install.php</exclude-pattern>
-		<!-- @todo remove test exclusion -->
-		<exclude-pattern>./tests/app/</exclude-pattern>
+		<exclude-pattern>.phtml</exclude-pattern>
```

Comment:
@Frenzie Do you know if it is possible to have different rules depending on the file extension?

Reply:
Aren't you just missing a `*` or two?

Reply:
I.e., either *.phtml or */*.phtml

Reply:
Ah right. Seems to work though :-O

Reply:
By the way, can we specify which file extensions to scan inside this file? e.g. https://github.com/squizlabs/PHP_CodeSniffer/issues/2486

Reply:
Fixed https://github.com/FreshRSS/FreshRSS/pull/3488/commits/6740c5cbbb894d20f6f9422af33ec245c311baac

Reply:
Might need a `$` at the end for correctness?

Reply:
Done in https://github.com/FreshRSS/FreshRSS/commit/02add3419277757a80f6785959e2389c2ab1ddbf with a few more changes, in particular for `.js`